src/panels/lovelace/cards/hui-error-card.ts

```diff
@@ -43,8 +43,8 @@ export class HuiErrorCard extends LitElement implements LovelaceCard {
     return css`
       :host {
         display: block;
-        background-color: #ef5350;
-        color: white;
+        background-color: var(--error-color);
+        color: var(--on-error-color, white);
```

Comment:
```suggestion
        color: var(--color-on-error, white);
```

Let's not be stubborn and just use Bram's suggestion. As it follows the Material Variable naming 

Ex. var(--mdc-theme-on-primary)